docs/kv-store-how-to-use-it.md

```diff
@@ -0,0 +1,47 @@
+---
+id: basic-kv-commands
+title: Basic KV Commands
+sidebar_label: Basic KV Commands
+---
+
+There are several commands you can run with the CLI to interact with your Key-Value Storage. You can see the entire list of commands by running `nim kv` in the commands prompt.
+
+In this document, we're going over a few commands you can use.
+
+### Display a list all of your keys.
+
+You can list all of your Key-Values.
+
+Command:
+
+```
+nim kv list
+```
+
+### Get the value of a key.
+
+You can get the value of a key in your command prompt.
+
+```
+nim kv get [keyName]
+```
+
+### Set a value for a key.
+
+Here's how you can change the value of a key in your command prompt. In this example, we'll be using a key that is holding onto a number called `counter`.
+
+```
+nim kv set counter 42
+>OK
```

Comment:
please double check the output. I don't think there's a leading `>` here or in the next command.